StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -1,15 +1,61 @@
 ﻿using StudentManagement.Console;
+bool menu = false;
+var students = new List<Student>();
+var student = new Student();
+do
+{
+
+
+    Console.WriteLine("1 -- Insert new Student\t");
+    Console.WriteLine("2 -- Show all students\t");
+    Console.WriteLine("Choose");
+    int option = int.Parse(Console.ReadLine());
+   
+
+
+    
+    switch (option)
+    {
+        case 1:
+            Console.WriteLine("Enter student name");
+            student.Name = Console.ReadLine();
+            Console.WriteLine("Enter student subname");
+            student.Surname = Console.ReadLine();
+            Console.WriteLine("What the student is studying");
+            student.Occupation = Console.ReadLine();
+            Console.WriteLine("Enter student id");
+            student.Id = int.Parse(Console.ReadLine());
+            Console.WriteLine("How old is student");
+            student.Age = int.Parse(Console.ReadLine());
+            students.Add(student);
+            break;
+        case 2:
+            foreach (Student studentList in students)
+            {
+                Console.WriteLine(studentList);
+            }
+            break;
+        default: break;
+    }
+
+    Console.WriteLine("{0} {1} studies {2}. His id is {3} and he is {4}", student.Name, student.Surname, student.Occupation, student.Id, student.Age);
+    Console.WriteLine("Back to Menu? y/n");
+    char c = char.Parse(Console.ReadLine());
```

Comment:
wouldn't it be better if you move it to swtich case and if user can leave the app if he presses a 0?